CalendarApp/EventKit Handlers/EKEventBuilder.h

```diff
@@ -0,0 +1,21 @@
+//
+//  EKEventBuilder.h
+//  CalendarApp
+//
+//  Created by Angelina Zhang on 7/19/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "CalendarApp-Swift.h"
+#import <EventKit/EventKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EKEventBuilder : NSObject
+
+- (Event *)getEventFromEKEvent:(EKEvent *)ekEvent;
+- (NSMutableArray<Event *> *)getEventsFromEKEventArray:(NSArray<EKEvent *> *)ekEvents;
```

Comment:
I tend to prefer plural over giving a collection type. `Events` vs `EventArray`. Because what if it changes later to an event queue or event linked list or some other type.